src/Raven.Server/RavenServer.cs

```diff
@@ -452,6 +460,12 @@ public double RemainingCpuCredits
                 set => Interlocked.Exchange(ref _remainingCpuCredits, value);
             }
 
+            public int ForceSync
+            {
+                get => Interlocked.CompareExchange(ref _forceSync, 0, 0);
```

Comment:
Huh? Why not `Interlocked.Read` ?

Reply:
Because Interlocked.Read doesn't work for int, just long.
This is infact like Interlocked.Read. It performs an atomic read operation with no writing.

Reply:
Does it matter? Do you want me to change the type to long and use Interlocked.Read?

Reply:
It mutates the data (more expensive. Can use `Thread.VolatileRead` instead, I think

Reply:
Few questions:
- why this is not bool?
- getting/setting bool is atomic
- why do we care here about interlocked/volatile/etc?

Reply:
Actually, we don't really care.
The reason I did this is to be consistent with the above RemainingCpuCredits variable where it's actually important. So I guess I'll just use a normal bool there.